addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -61,7 +61,9 @@ var MassMailingFieldHtml = FieldHtml.extend({
         }
 
         var $editable = this.wysiwyg.getEditable();
-
+        if (this.wysiwyg.snippetsMenu) {
+            await this.wysiwyg.snippetsMenu.cleanForSave();
+        }
```

Comment:
This entire commit seems like an important fix to do in stable versions if it is actually needed. Can you verify and move it to another PR if this is the case?

Reply:
I think this is safe to do in stable versions (ignore < 13.0) to handle potential custo or community modules.

Reply:
I checked the code but it seems that in [14.0 - 14.2] we can't get the `this.wysiwyg.snippetsMenu` object (to run its `cleanForSave` method) in fieldHtml code. Is this intended to be this way ?

Reply:
Ok, if code different in the old editor, we can care about fixing that in stable versions later.